crates/collab_ui/src/chat_panel.rs

```diff
@@ -335,67 +336,77 @@ impl ChatPanel {
         };
         let this = cx.view().clone();
 
-        v_flex()
-            .w_full()
-            .relative()
-            .overflow_hidden()
-            .when(!is_continuation_from_previous, |this| {
-                this.pt_3().child(
-                    h_flex()
-                        .text_ui_sm()
-                        .child(
-                            div().absolute().child(
+        let mentioning_you = message
+            .mentions
+            .iter()
+            .any(|m| m.1 == self.client.user_id().unwrap());
+
+        v_flex().w_full().relative().child(
+            div()
+                .bg(if mentioning_you {
+                    Fill::from(cx.theme().colors().background)
+                } else {
+                    Fill::default()
+                })
+                .p_1()
+                .rounded_md()
```

Comment:
Just curious - how does this look when two consecutive message both mention you?

Reply:
Now it looks like this:
<img width="242" alt="Screenshot 2024-01-29 at 22 30 38" src="https://github.com/zed-industries/zed/assets/62463826/f3c22388-4711-4a1e-ac90-d39a362c9d27">

Reply:
Haven't tested that one:
<img width="241" alt="Screenshot 2024-01-29 at 22 33 52" src="https://github.com/zed-industries/zed/assets/62463826/216e1889-d363-4251-baad-7e6a8e1b1179">

When there is a message in between, it looks nice:
<img width="239" alt="Screenshot 2024-01-29 at 22 33 05" src="https://github.com/zed-industries/zed/assets/62463826/5db743b5-19c6-4c2e-bd3b-443429917488">

Reply:
I changed it locally to this. But one note, this brings a little inconsistency with spacing between messages that do/not have a mention inside the message. It's not much, but I'm not sure if this is a problem.
<img width="241" alt="Screenshot 2024-01-29 at 22 39 46" src="https://github.com/zed-industries/zed/assets/62463826/7bef56bb-af75-436f-a026-eb001a07e4f5">

Reply:
I fixed that inconsistent spacing issue in my latest commit.